src/libraries/System.Formats.Asn1/Directory.Build.props

```diff
@@ -0,0 +1,6 @@
+﻿<Project>
+  <Import Project="..\Directory.Build.props" />
+  <PropertyGroup>
+    <StrongNameKeyId>Open</StrongNameKeyId>
```

Comment:
For what is worth, this is the default key value of the repo, so you could get rid of this file if you wanted.

https://github.com/dotnet/runtime/blob/50e15b2a76d153ee3511de4b72bf685595a55121/src/libraries/Directory.Build.props#L179